css/style.css

```diff
@@ -356,3 +370,92 @@ div .taggs {
   padding: 0 0.5rem;
   margin: 1em;
 }
+
+/* Contact page */
+.contact {
+  width: 100%;
+  height: 740px;
+  background: #6070ff;
+  background-image: url(/images/contact/layers.svg);
+  background-size: 600px 750px;
+  background-repeat: no-repeat;
+  background-position-x: -70px;
+  background-position-y: 10px;
+  border-top-left-radius: 6rem;
+  padding-bottom: 4rem;
+}
+
+.contact-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  margin-top: 80px;
+  padding-top: 2.5rem;
+  margin-bottom: 0;
+}
+
+.contact-content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  text-align: center;
+  color: #ebebff;
+  padding: 0 1.5rem;
+}
+
+.my-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1.5rem;
+  gap: 1.3rem;
+}
+
+.inputField {
+  padding: 0.9rem;
+  border-radius: 10px;
+  border: none;
+  font-style: normal;
+
+  /* font-family: 'Poppins', sans-serif; */
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.inputField:hover {
+  transform: translate(5px);
+}
+
+.textareaField {
+  border-radius: 10px;
+  padding: 0.9rem;
+  font-style: normal;
+
+  /* font-family: 'Poppins', sans-serif; */
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  height: 170px;
+}
+
+.textareaField:hover {
+  transform: translate(5px);
+}
+
+.my-button {
+  align-items: center;
+  padding: 12px;
+  background: #fff;
+  border-radius: 8px;
+  color: #6070ff;
+  width: 50%;
+  border: none;
+
+  /* font-family: 'Poppins', sans-serif; */
```

Comment:
- [ ] [OPTIONAL] Kindly remove this used line of code to keep your work consistently clean and readable. It is a standard practice to remove unnecessary codes or those that we commented out.